custodia/store/sqlite.py

```diff
@@ -110,8 +110,11 @@ def list(self, keyfilter=''):
                     continue
                 if not row[0].startswith(child_prefix):
                     continue
-                value.append(row[0][len(child_prefix):].lstrip('/'))
-
+                result = row[0][len(child_prefix):].lstrip('/')
+                if not row[1]:
```

Comment:
As we discussed on the #152, this value will not be Null, but it will be empty for this case, so I think that we have to keep with the current statement.